scripts/push-docker-images

```diff
@@ -76,7 +76,15 @@ if [[ $MANIFEST == "true" ]]; then
     fi
     cat <<< $(jq '.+{"experimental":"enabled"}' $DOCKER_CLI_CONFIG) > $DOCKER_CLI_CONFIG
     echo "Enabled experimental CLI features to create the docker manifest"
-    docker manifest create $IMAGE_REPO:$VERSION $MANIFEST_IMAGES
+
+    manifest_create_args=($IMAGE_REPO:$VERSION $MANIFEST_IMAGES)
+    manifest_exists=$(docker manifest inspect $IMAGE_REPO:$VERSION > /dev/null ; echo $?)
+    # manifest already exists; therefore, amend is required
+    if [[ manifest_exists -eq 0 ]]; then
+      manifest_create_args+=(--amend)
```

Comment:
Docker CLI has an open issue ([#954](https://github.com/docker/cli/issues/954)) for the `--amend` flag.  Is this script impacted?

Reply:
saw that as well, but I still saw expected behavior when I tested this locally using my own dockerhub repo. I also do `docker manifest create --amend` first, then proceed with `docker manifest annotate` and finally `docker manifest push` which is a little different from what that guy was doing so possibly not the same issue.